contents/blog/persons-on-events.md

```diff
@@ -0,0 +1,34 @@
+---
+date: 2022-08-29
+title: 'How we’re improving performance by combining persons and events’'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+In [our last product update](/blog/the-posthog-array-1-39-0) we announced a beta for a substantial change to the way we handle persons and events on PostHog. Today, after gathering your feedback and seeing the improvements to query performance, we’re rolling this change out to all PostHog users. 
+
+This change essentially combines [persons](/manual/persons) and [events](/manual/events) into a single ClickHouse table, moving person IDs and properties _onto_ events. You won’t see any UI changes — persons will still have their own Persons & Groups section on the sidebar, for example — and there’ll be no loss of functionality, but here are some changes you may see:
+
+- **Faster results for queries involving person properties and events**. Putting persons and events into a single ClickHouse table means we no longer have to join tables to get results on queries involving these data. As a result, query performance will improve by up to 400%.
+
+- **Faster filtering of events with person properties**. Similarly, filtering events by person properties is much faster when a join is no longer necessary. Anywhere where you’re working with persons and events, PostHog will be faster. 
```

Comment:
My tweak seems to suggest it's only true when all 3 are present, which isn't the case. Hmmm.

```suggestion
- **Faster filtering of events with person properties**. Similarly, filtering events by person properties is much faster when a join is no longer necessary. Anywhere where you’re working with persons, groups, and events, PostHog will be faster. 
```